### ISSUE TITLE
Make shortcuts in storybook optionable or move to addon

### ISSUE THREAD

**Issue by Kiwka** [storybooks/storybook#3196]
### Issue details

On one of the components that we have in our storybook, we have several shortcuts that we want add functionality to. One of them overlaps with storybook default one `⌘ ⇧ C`, and disable us to handle this shortcut properly on the component.

My proposal is to add the possibility to enable or disable shortcuts in storybook through `addon-options`, keeping them as enabled by default.

**Comment by PunitGr** [comment]
Hello @Kiwka @igor-dv 👋 

I'd love to take this up!

Does this spec seem fine to you folks?

```
setOptions({
    ...
    enableShortcuts: false, // true by default
})
```

**Comment by Hypnosphi** [comment]
Sure @PunitGr go for it

**Comment by Kiwka** [comment]
Thanks @PunitGr for looking into this 🙇‍♀️

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.0`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.0)
